deal.II-toolchain/packages/petsc.package

```diff
@@ -98,6 +98,7 @@ package_specific_setup () {
 
 package_specific_register () {
     export PETSC_DIR=${INSTALL_PATH}
+    export SCALAPACK_DIR=${INSTALL_PATH}
```

Comment:
may you please encapsulate this as follows:
```
if [ ! -z "${PARMETIS_DIR}" ]; then
    export SCALAPACK_DIR=${INSTALL_PATH}
fi
```
since scalapack is only installed by petsc in this case